app/src/main/java/org/mozilla/focus/compose/Modifier.kt

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.compose
+
+import androidx.compose.runtime.Stable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.composed
+import androidx.compose.ui.draw.scale
+import androidx.compose.ui.platform.LocalLayoutDirection
+import androidx.compose.ui.unit.LayoutDirection
+
+/**
+ * Modifier used for performing auto mirroring in RTL for composable.
+ */
+@Stable
+fun Modifier.autoMirroringModifier(): Modifier = composed {
```

Comment:
I think most of the current modifiers are named to express an action that will be executed or property that is updated for the Composable without the `Modifier` suffix.
Can this be just `autoMirror`?